### PR TITLE
conversion to Cobra commands

### PR DESCRIPTION
**Pull request by bhamail**
Use Cobra commands for CLI parsing/processing.

Probably not perfect, but at least avoids scattered `os.Exit()` and similar `abEnd` exit paths, which should make things easier to test, etc.

Might be able to reuse the `ErrorExit` custom error approach from the Nancy `exit-purge` work.

Open to suggestions!

### CODE REVIEW

**Comment by bhamail** [comment]
Whoa! Please avert your eyes, while I fix my approach! I just realized how I can pass in the pointer to `config` elements instead of what I'm doing now....

**Comment by bhamail** [comment]
If nobody objects, I'd like to YOLO this and merge it soon.

File: cmd/fry.go [hunk 1]
```diff
@@ -0,0 +1,98 @@
+/*
+Copyright © 2020 NAME HERE <EMAIL ADDRESS>
```
Comment (DarthHater) [inline cmd/fry.go]: lol HELLO NAME HERE!

File: cmd/fry.go [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+	"github.com/sonatype-nexus-community/nancy/cyclonedx"
+	"github.com/spf13/cobra"
+)
+
+// fryCmd represents the fry command
+var fryCmd = &cobra.Command{
+	Use:   "fry",
+	Short: "Cook up some hashes",
+	Long: `Explain how we fry up some hashes, in great detail.
```
Comment (DarthHater) [inline cmd/fry.go:19]: LOOOOOOL

File: cmd/fry.go [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+	// and all subcommands, e.g.:
+	fryCmd.PersistentFlags().StringVar(&config.Path, "path", "", "Path to file with sha1s")
+	fryCmd.PersistentFlags().StringVar(&config.User, "user", "admin", "Specify Nexus IQ username for request")
+	fryCmd.PersistentFlags().StringVar(&config.Token, "token", "admin123", "Specify Nexus IQ token/password for request")
+	fryCmd.PersistentFlags().StringVar(&config.Server, "server-url", "http://localhost:8070", "Specify Nexus IQ Server URL")
+	fryCmd.PersistentFlags().StringVar(&config.Application, "application", "", "Specify application ID for request")
+	fryCmd.PersistentFlags().StringVar(&config.Stage, "stage", "develop", "Specify stage for application")
+	fryCmd.PersistentFlags().IntVar(&config.MaxRetries, "max-retries", 300, "Specify maximum number of tries to poll Nexus IQ Server")
+
+	// Cobra supports local flags which will only run when this command
```
Comment (DarthHater) [inline cmd/fry.go]: You might remove these stock comments

File: cmd/root.go [hunk 4]
```diff
@@ -0,0 +1,94 @@
+/*
+Copyright © 2020 NAME HERE <EMAIL ADDRESS>
```
Comment (DarthHater) [inline cmd/root.go]: NAME HERE IT IS NICE TO MEET YOU

File: types/types.go [hunk 5]
```diff
@@ -1,10 +1,23 @@
 package types
 
+/*type LogLevel int
```
Comment (DarthHater) [inline types/types.go]: May as well remove these unless you are like OMG I WILL FIGURE THIS OUT

File: cmd/fry.go (hunk 3 above)
Comment (bhamail) [reply cmd/fry.go]: > You might remove these stock comments

Done  40c2917

File: types/types.go (hunk 5 above)
Comment (bhamail) [reply types/types.go]: > May as well remove these unless you are like OMG I WILL FIGURE THIS OUT

Done 40c2917

File: cmd/fry.go (hunk 1 above)
Comment (bhamail) [reply cmd/fry.go]: > lol HELLO NAME HERE!

Doh! a9af601

File: cmd/root.go (hunk 4 above)
Comment (bhamail) [reply cmd/root.go]: > NAME HERE IT IS NICE TO MEET YOU

Doh! a9af601

**Pull request merged**